src/gsIO/gsFileManager.h

```diff
@@ -18,93 +18,171 @@
 namespace gismo
 {
 
-/// @brief This class checks if the given filename can be found
-///  in one of the pre-defined search paths. It is possible to
-///  register additional search paths.
+/// @brief File-system related functionality.
 ///
+/// Input paths to the fuctions of this class can be given using any
+/// valid path seperator, this is in Unix only "/" and in Windows both
+/// "/" and "\\".
+///
+/// Return values only contain the preferred native path separator, which
+/// is in Unix "/" and in Windows "\\".
+///
+/// Output Paths always end with path separator, files without.
+///
+/// Does not check for special system cases, like AUX under Windows.
+/// (see https://en.wikipedia.org/wiki/Filename)
 /// @ingroup IO
 class GISMO_EXPORT gsFileManager
 {
 public:
 
-    /// Get native path seperator
-    static char getNativePathSeparator();
+    /// Get preferred native path seperator
+    static const char getNativePathSeparator();
 
-    /// Checks if the path is fully qualified
-    /// If a name starts with "/", it is considered fully qualified
+    /// Get valid path seperators
+    static const std::string& getValidPathSeparators();
+
+    /// Get system depending not allowed characters for paths and filenames
```

Comment:
rewrite as:
/// Get system-dependent invalid characters for paths and filenames

Reply:
done